packages/react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -494,10 +494,10 @@ class DropdownInternal extends React.Component<IDropdownInternalProps, IDropdown
   };
 
   /** Get either props.placeholder (new name) or props.placeHolder (old name) */
-  private get _placeholder(): string | undefined {
+  private _getPlaceholder = (): string | null => {
```

Comment:
Can you add a comment about why this needs to return explicit null instead of undefined?

Reply:
It turns out it wasn't necessary to return null instead of undefined; changed it back to return undefined.